### PR TITLE
Corrections on Readme

### PR DESCRIPTION
**Pull request by SuperKali**
I made corrections on the specifications of the router itself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 - Network support: 4G LTE Networks TDD/FDD + WCDMA
 - Chipset: Qualcomm SDX24 Platform
 - CPU: 1x Cortex A7 up to 1.4GHz
 - RAM: 256MB
 - NAND: 512MB
-- 4G LTE Cat: 20
-- Network connectivity: 2.5GbE interface with PoE 802.3af/at (bundled injector)
+- 4G LTE Cat: 20 downlink & 13 uplink
+- 4G LTE Speed: 2Gbit downlink & 316Mbit uplink
+- Network connectivity: 2x1GbE interface
```
Comment (stich86) [inline README.md]: Please remove it, the readme is only for module itself not the router board

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

**Review by stich86 — APPROVED** [review]

File: swver.md
Comment (ArturDorochowicz) [inline swver.md:1]: Hi @stich86 

Not sure if this is useful, but I'll share anyway.

I have this router, now running OpenWrt (thank you!). I bought it second hand, it apparently came from some Finnish telecom called DNA.

Not sure if the table here lists router software versions or modem firmware versions. `ATI` command for my modem returns:
```
Manufacturer: ZTE CORPORATION
Model: MF289F
Revision: BD_DNAFIMF289F1V1.0.0B11
[...]
```
(and router software version ended in B12)

Router hardware revision is I think what is called T2 - based on the fact that it has two IPQ4019 radios.

Cheers!

Comment (stich86) [reply swver.md:1]: Oh really nice.

Do you want to take a dump and share it?
I can ask to add it to Mega folder

Just need dumps from:

Boot
Sysfs
NON-HLOS
Uefi

and `config` file from EFS

Let me know :)

Comment (ArturDorochowicz) [reply swver.md:1]: I can, but I'll need instructions to follow. I don't know much about this stuff.

Comment (stich86) [reply swver.md:1]: you have istruction here to dump whole module (or single partition) using EDL tools: https://github.com/stich86/ZTE-MF289F-Recovery/blob/main/edl.md

needs a Linux machine (or live from https://github.com/bkerler/edl) and modem seated into an mPCIe-to-USB adapter.

Alternative way is to dump partition from module itself and the copy it from ADB (install as package on OpenWRT), something like that:

```
adb shell
cat /dev/mtd13 > /tmp/uefi
```

Exit from the shell and do: `adb pull /tmp/uefi uefi.elf`

```
adb shell
rm /tmp/uefi
cat /dev/mtd16 > /tmp/boot
```

Exit from the shell and do: `adb pull /tmp/boot boot.img`

```
adb shell
rm /tmp/boot
cat /dev/mtd18 > /tmp/modem
```

Exit from the shell and do: `adb pull /tmp/modem modem.img`

```
adb shell
rm /tmp/modem
cat /dev/mtd27 > /tmp/system
```

Exit from the shell and do: `adb pull /tmp/system system.img`
`adb shell rm /tmp/system` -- to clean up last file

Collect those files from your router via SCP\SFP

Regarding `config` file,  if you have module still on your router, install `ser2net`, expose port `ttyUSB0` and connect it using QPST (check my guide [here](https://forum.fibra.click/d/32421-zte-mf289f-vodafone-fwa-sblocco-aggregazione-b20-su-modello-vfit)  for B20 prune), launch `EFS Explorer` and copy it from the root 

When you have all those files, share a link where you have uploaded them, don't post it here any dump, just link please.

Thanks :)

Comment (ArturDorochowicz) [reply swver.md:1]: Ok, I've got it all.

Comment (stich86) [reply swver.md:1]: good when you have them let me know ;)

Comment (ArturDorochowicz) [reply swver.md:1]: @stich86  Yes, that's what I meant, I have the files retrieved.

Let me know when you have it.

Comment (stich86) [reply swver.md:1]: got it, you can also remove link and password ;)

just a question: is the RJ-11 port works? VoLTE?

So I can update swver and VoLTE info

Thanks!

Comment (ArturDorochowicz) [reply swver.md:1]: > just a question: is the RJ-11 port works? VoLTE?

I don't know. I don't think I have gear to check this.

Comment (stich86) [reply swver.md:1]: > > just a question: is the RJ-11 port works? VoLTE?
> 
> I don't know. I don't think I have gear to check this.

Hi mate,

firmware was repacked and uploaded into external share.
I've tested it and also update CA combo. Looks like it doesn't support VoLTE\Voice port as default, so need to swap with Vodafone DE firmware to make it works

Thank you so much for sharing!

**Pull request merged**